lib/hella-redis/connection_pool.rb

```diff
@@ -8,15 +8,13 @@ module HellaRedis
   class ConnectionPool
     def initialize(config)
       @pool = ::ConnectionPool.new({
-        :timeout => config.timeout,
-        :size    => config.size
+        timeout: config.timeout,
+        size: config.size,
       }) do
-        ::Redis::Namespace.new(config.redis_ns, {
-          :redis => ::Redis.new({
-            :url    => config.url,
-            :driver => config.driver
-          })
-        })
+        ::Redis::Namespace.new(
+          config.redis_ns,
+          redis: ::Redis.new(url: config.url, driver: config.driver),
+        )
```

Comment:
Can you do our line wrapping convention here please? E.g. break after the `@pool =`.